packages/x-adapter-platform/src/schemas/models/related-prompt.schema.ts

```diff
@@ -0,0 +1,13 @@
+import { createMutableSchema } from '@empathyco/x-adapter';
+import { RelatedPrompt } from '@empathyco/x-types';
+import { PlatformRelatedPrompt } from '../../types/models/related-prompt.model';
+
+/**
+ * Default implementation for the RelatedPromptSchema.
+ */
+export const relatedPromptSchema = createMutableSchema<PlatformRelatedPrompt, RelatedPrompt>({
+  modelName: () => 'RelatedPrompt',
+  nextQueries: () => [],
+  suggestionText: 'suggestionText',
+  type: () => 'SYNTHETIC'
```

Comment:
why are we mapping an empty array instead of the actual nextQueries from the response? Same for the type, I think we should not map it always as `SYNTHETIC` 🤔

Reply:
Fixed.